### ISSUE TITLE
I trained the model based on the C3D dataset. The average L2 chamfer distance is 8.5028^(-3). The pre-trained model you provided is 8.7199^(-3), which is worse than the data 7.60*10^(-3) in the paper. But the results of other datasets are the same with the paper's result. Except for the dataset path, I did not modify your code. Is this result correct?

### ISSUE THREAD

**Issue by zzczzc-0118** [AllenXiangX/SnowflakeNet#37]

**Comment by xiaodu510** [comment]
Hello, did you mean that the replicated Snowflakenet is the same as the PCN dataset on other datasets as stated in the paper? When I tried to replicate the SPD model on the PCN dataset, the accuracy was 7.21 when using the pre-trained weights provided by the author, and 7.40 when training from scratch. After 359 rounds, the accuracy became NAN. I suspect it might be due to gradient explosion. I would like to ask about the conditions you used for the replication. I will retrain it.
